src/main/java/sirius/db/mixing/properties/MultiPointLocationProperty.java

```diff
@@ -101,18 +100,16 @@ public Object transformFromDatasource(Class<? extends BaseMapper<?, ?, ?>> mappe
                     "MultiPointLocationProperty currently only supports Mango as mapper!");
         }
 
-        List<Tuple<Double, Double>> locations = new ArrayList<>();
-
         if (!object.isNull()) {
             Object coordinates = ((Document) object.get()).get("coordinates");
             if (coordinates instanceof List<?>) {
-                ((List<List<Double>>) coordinates).stream()
-                                                  .map(entry -> Tuple.create(entry.get(0), entry.get(1)))
-                                                  .collect(Lambdas.into(locations));
+                return ((List<List<Double>>) coordinates).stream()
+                                                         .map(entry -> Tuple.create(entry.get(0), entry.get(1)))
+                                                         .collect(Collectors.toList());
```

Comment:
This means the returned list can not be altered afterwards, right? Is that okay in this case?

Reply:
stream.toList() is immutable / stream.collect(Collectors.toList()) is an array list

Reply:
Ahhhh, thanks +1 good to know